client/client-multi/client-c/src/kaa/kaa_logging.c

```diff
@@ -70,6 +70,15 @@ struct kaa_log_collector {
 static const kaa_service_t logging_sync_services[1] = {KAA_SERVICE_LOGGING};
 
 
+kaa_error_t kaa_logging_need_logging_resync(kaa_log_collector_t *self, bool *result)
+{
+    KAA_RETURN_IF_NIL2(self, result, KAA_ERR_BADPARAM);
+    if (self->is_sync_ignored) {
+            return KAA_ERR_NONE;
```

Comment:
I think the result variable should be initialized explicitly in both cases. Please, also fix code alignment.